README.md

```diff
@@ -1,7 +1,9 @@
-> gh-install: Install pre-built binaries from github
+# gh-install
 
 [![Build Status](https://dev.azure.com/crate-ci/crate-ci/_apis/build/status/gh-install%20v1?branchName=master)](https://dev.azure.com/crate-ci/crate-ci/_build/latest?definitionId=8&branchName=master)
 
+Install pre-built binaries from GitHub.
+
```

Comment:
```suggestion
# gh-install

> Install pre-built binaries from GitHub.

[![Build Status](https://dev.azure.com/crate-ci/crate-ci/_apis/build/status/gh-install%20v1?branchName=master)](https://dev.azure.com/crate-ci/crate-ci/_build/latest?definitionId=8&branchName=master)

```

Reply:
I knew, I knew! The description has to be styled.